pkg/services/ngalert/eval/eval.go

```diff
@@ -564,8 +567,6 @@ func (evalResults Results) AsDataFrame() data.Frame {
 		labelColumns = append(labelColumns, k)
 	}
 
-	labelColumns = sort.StringSlice(labelColumns)
```

Comment:
This changes the structure of the data frame, and can change the table display after each evaluation? Why should we remove it?

Reply:
> This changes the structure of the data frame, and can change the table display after each evaluation

can you please elaborate?

I removed it because the static analysis highlighted it as a useless type conversion. Looking at usages below I agreed with it.

Reply:
Sorry - I presumed it was `func StringSlice(s []string)` instead of `func (s StringSlice) Sort()` - a call instead of a type conversion.